src/world.hpp

```diff
@@ -72,6 +72,7 @@ class World
 	vec2 lanes[4];
 
 	Mix_Music* m_background_music;
+	Mix_Chunk* m_roy_whistle;

```

Comment:
What's the difference between `Mix_Music` and `Mix_Chunk`? (This isn't a problem with the PR, I'm just curious)

Reply:
Music is 1 track only, Chunk can be used for multiple tracks.